index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="index.css" />
+</head>
+<body>
+    <main>
+        <header class="header">
+            <a href="#">My Logo</a>
+            <nav><img src="Icons/Union.png" alt="burger icon" /></nav>
+        </header>
+        <section class="content">
+            <div class="gretting-text">
+                <h1>Hey There.<br/>
+                    I’m Magor</h1>
+            </div>
+            <div class="description">
+                <h2>I’am a Software Developer</h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+            <ul class="social-networks-block">
+                <li><a href="#"><img src="Icons/github.png" alt="burger icon" /></a></li>
+                <li><a href="#"><img src="Icons/Linkedin.png" alt="burger icon" /></a></li>
+                <li><a href="#"><img src="Icons/Vector1.png" alt="burger icon" /></a></li>
+                <li><a href="#"><img src="Icons/twitter.png" alt="burger icon" /></a></li>
+                <li><a href="#"><img src="Icons/m.png" alt="burger icon" /></a></li>
+            </ul>
```

Comment:
- [ ] Good job implementing some changes required by the previous reviewer. As mentioned by the previous reviewer, I noticed you are still having one broken image on your icons images. Pease makes sure you fix that to match the Figma design.
Here is what your social links images look like:
![Screenshot from 2022-06-30 21-37-57](https://user-images.githubusercontent.com/83042742/176763052-5be65f80-54b6-4641-86c2-bd52ee652072.png)